library/azure_rm_devtestlabvirtualmachine.py

```diff
@@ -0,0 +1,516 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_devtestlabvirtualmachine
+version_added: "2.8"
+short_description: Manage Azure DevTest Lab Virtual Machine instance.
+description:
+    - Create, update and delete instance of Azure DevTest Lab Virtual Machine.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: True
+    lab_name:
+        description:
+            - The name of the lab.
+        required: True
+    name:
+        description:
+            - The name of the virtual machine.
+        required: True
+    notes:
+        description:
+            - The notes of the virtual machine.
+    os_type:
+        description:
+            - The OS type of the virtual machine.
+    size:
```

Comment:
any choice?

Reply:
that's VM size so i guess list would be too long, and it's changing.
that's exactly the same option as **vm_size** from **azure_rm_virtualmachine**.
btw, perhaps we shoudl rename it to **vm_size**

Reply:
renamed to **vm_size** to added same description